backend/src/main/java/com/htecgroup/SkyNest/service/impl/EmailServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.htecgroup.SkyNest.service.impl;
+
+import com.htecgroup.SkyNest.service.EmailService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+@Service
+public class EmailServiceImpl implements EmailService {
+  @Autowired private JavaMailSender javaMailSender;
```

Comment:
Pls discuss internally the two approaches, using @Autowire (not advisable :) ) or using controller injection using Lombok. I saw few different PRs with different approaches.. it is strongly advisable to have same code standard,
@DarkoVeljanoskiHtec @MarkoRadunovicHtec @ivanfhtec @DraganMitrasinovicHtec same applies for you all